v1/types/types.gunk

```diff
@@ -252,6 +252,26 @@ type Location struct {
 	Longitude float64 `pb:"2" json:"longitude"`
 }
 
+type OfflineUserInfo struct {
+	// UserID
+	UserID string `pb:"1" json:"user_id"`
+
+	// FirstName of the person
+	FirstName string `pb:"2" json:"first_name"`
+
+	// MiddleName or middle names (space separated) of the person
+	MiddleName string `pb:"3" json:"middle_name"`
+
+	// LastName of person
```

Comment:
If you say "or middle names", might as well say "or last names" here too. Don't Filipinos have two surnames too, due to the Spanish influence?

Reply:
True.  Changed